openff/toolkit/tests/test_molecule.py

```diff
@@ -3724,6 +3724,14 @@ def test_visualize_nglview(self):
         # Ensure an NGLView widget is returned
         assert isinstance(mol.visualize(backend="nglview"), nglview.NGLWidget)
 
+        # Providing other arguments is an error
```

Comment:
Non-blocking: I'd like these errors (which I think I may have originally written) to not be `ValueError` and instead be something custom to the toolkit. Since #1021 isn't merged yet, it doesn't need to happen here, and I've added some TODOs to note it.